### PR TITLE
Split props for base and remix links

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1120

Here refactored links components to generate separate props for base and remix variants.
Removed prefetch from base links.

Added following props to remix links

- reloadDocument: boolean
- replace: boolean
- preventScrollReset: boolean

Now all components consistently have meta and props

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/sdk-components-react-remix/src/link.ws.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      type: "string",
+      control: "url",
+      required: false,
+    },
+  },
+  initialProps: [
+    "href",
+    "target",
+    "prefetch",
+    "prefetch",
```
Comment (kof) [inline packages/sdk-components-react-remix/src/link.ws.ts:19]: twice

Comment (kof) [reply packages/sdk-components-react-remix/src/link.ws.ts:19]: we need to somehow enforce this props list through type

**Review by kof — CHANGES_REQUESTED** [review]
prefetch is missing now by default in properties panel, is it missing in initialProps?

**Comment by TrySound** [comment]
Fixed, forgot to reexport props. Need to generate these entry points.

**Pull request merged**